resources/authgear/templates/en/web/authflowv2/layout.html

```diff
@@ -0,0 +1,5 @@
+<!-- This template can be customized by each app. -->
+<!-- Customized layout.html must be placed in templates/en/web/authflowv2/layout.html of app resources -->
+<div class="layout--default">
+  {{ template "widget" . }}
```

Comment:
Should be able to pass `Classname` (to control flex-grow, etc) to widget.

Reply:
I think we don't want outside to control the class of `widget`, because it is possible to break the ui by applying incorrect styles. For example, applying "flex" to widget which is original a "block" will break the whole ui.

To enforce encapsulation of the widget, I prefer we don't allow giving any additional class to widget. If we need to style it for layout purpose, we will do it in the layout css.

```css
.layout .widget {
  flex: 1 0 auto;
}
```